src/yuzu/main.cpp

```diff
@@ -2999,6 +3000,12 @@ int main(int argc, char* argv[]) {
     chdir(bin_path.c_str());
 #endif
 
+#ifdef __linux__
+    // Set the DISPLAY variable in order to open web browsers
```

Comment:
```suggestion
    // Set the DISPLAY variable in order to open web browsers
    // TODO (lat9nq): Find a better solution for AppImages to start external applications 
```
Can you add this comment that points at me? (Making a suggestion so it's easier to add with GitHub)